SoftLayer/CLI/block/order.py

```diff
@@ -49,8 +46,8 @@
               'space along with endurance block storage; specifies '
               'the size (in GB) of snapshot space to order')
 @click.option('--service-offering',
-              help='The service offering package to use for placing '
-              'the order [optional, default is \'storage_as_a_service\']',
+              help="""The service offering package to use for placing the order.
+[optional, default is \'storage_as_a_service\']. enterprise and performance are depreciated""",
```

Comment:
I think this looks good, but I noticed it is only here in the block volume order command. Would it be possible to add it to SoftLayer/CLI/file/order.py as well?